runtime/src/mem/jemalloc/mem-jemalloc.c

```diff
@@ -47,11 +47,12 @@
 // the performance difference in a playground just to see if it's worth
 // investigating further.
 
-// TODO put these into a struct?
-static void*  heap_base = NULL;
-static size_t heap_size = 0;
-static size_t cur_heap_offset = 0;
-static pthread_mutex_t chunk_alloc_lock;
+static struct shared_heap {
+  void* base;
+  size_t size;
+  size_t cur_offset;
+  pthread_mutex_t alloc_lock;
+} heap; // static, will be "zero" initialized automatically
```

Comment:
POSIX defines PTHREAD_MUTEX_INITIALIZER to statically initialize a pthread_mutex_t.  It would be superfluous here because we're doing an explicit pthread_mutex_init() on it, but it wouldn't hurt anything to use it.